src/components/start/startmenu.scss

```diff
@@ -9,7 +9,8 @@
 .dskApp {
   margin: 4px;
   padding: 12px;
-  height: 66px;
+  height: 86px;
+  width: 76px;
```

Comment:
It looks way too wide for normal 1920x1080 resolution

![image](https://user-images.githubusercontent.com/54899800/142867933-da7ad7ee-8193-44cd-b7d1-ba620a099f9c.png)

we can fit a whole icon down there

Reply:
> It looks way too wide for normal 1920x1080 resolution

yes it does , but donno why coz I am taking values from real win11 and it looks fine in win11 :(

will look into it :)

Reply:
Yes sure, remember tha values can mean differ for an os and a web application.

Reply:
donno why in my laptop it is looking fine (1920x1080 resolution , 125% scaling)

|![image](https://user-images.githubusercontent.com/89068816/142881730-0e8f7a4f-f690-4839-b7cf-c53d17af2d8e.png)|![image](https://user-images.githubusercontent.com/89068816/142882063-55e39d92-378f-4aad-a69b-884dc95160af.png)|![image](https://user-images.githubusercontent.com/89068816/142882189-d451d99b-3e38-4525-80c0-fa3dcdca3f6b.png)|
|---|---|---|
| win11.blueedge.me | real win11 | 218.pr.blueedge.me |

Reply:
can you check once again, i removed the padding (it is not there in real win11)

Reply:
> donno why in my laptop it is looking fine (1920x1080 resolution , 125% scaling)
> 
> 
> 
> |![image](https://user-images.githubusercontent.com/89068816/142881730-0e8f7a4f-f690-4839-b7cf-c53d17af2d8e.png)|![image](https://user-images.githubusercontent.com/89068816/142882063-55e39d92-378f-4aad-a69b-884dc95160af.png)|![image](https://user-images.githubusercontent.com/89068816/142882189-d451d99b-3e38-4525-80c0-fa3dcdca3f6b.png)|
> 
> |---|---|---|
> 
> | win11.blueedge.me | real win11 | 218.pr.blueedge.me |
> 
> 
> 
> 
> 
> hmm so I checked in my fathers laptop it is looking just like the screenshot u sent though it is looking fine in my laptop even though both the laptops are 1920x1080 resolution and scaling set to 125%

It may be the scalding hat makes it look like the padding is "non-existent".

Reply:
> It may be the scalding hat makes it look like the padding is "non-existent".

donno about that coz in two different laptops both 1080p screens both 125% scaling in one it looked fine in another it didn't (only difference being one had a 15.6inch screen and the other 14inch screen)

Reply:
@blueedgetechno ig it should be fine now (now it looks fine in the other laptop too) it was the padding then ig

Reply:
wait do not merge found one more thing

Reply:
yes it is